script/script.js

```diff
@@ -542,7 +540,7 @@ const categoriesDatatable = {
             header: "Film's category",
             editor: "text",
             fillspace: true,
-            sort: "string"
+            sort: "text"
```

Comment:
it's no sence to use "text" mode for cells without collection or some complex template.
text mode requires checking cell template while sorting, byt we do not nned this action